bin/hisensetv/__main__.py

```diff
@@ -88,26 +90,28 @@ def main():
     )
 
     args = parser.parse_args()
-
-    if args.verbose:
-        level = logging.DEBUG
-    else:
-        level = logging.INFO
-
+    
+    startingpoint1 = '/usr/local/lib/node_modules/homebridge-hisense-tv-remotenow/bin/Hisensecerts/rcm_certchain_pem.cer'
+    startingpoint2 = '/usr/local/lib/node_modules/homebridge-hisense-tv-remotenow/bin/Hisensecerts/rcm_pem_privkey.pkcs8'
```

Comment:
I don't think this is a good solution. I understand that we need to pass the full path for this to work, but there's gotta be a better way. Maybe it is possible to get the `node_modules` directory in some way?